win32ss/user/ntuser/class.c

```diff
@@ -2679,6 +2701,32 @@ NtUserSetClassLong(HWND hWnd,
     return Ret;
 }
 
+ULONG_PTR
+APIENTRY
+NtUserSetClassLong(
+    _In_ HWND hWnd,
+    _In_ INT Offset,
+    _In_ ULONG dwNewLong,
+    _In_ BOOL Ansi)
+{
+    return IntNtUserSetClassLongPtr(hWnd, Offset, dwNewLong, Ansi, sizeof(LONG));
```

Comment:
```suggestion
    return IntNtUserSetClassLongPtr(hWnd, Offset, dwNewLong, Ansi, sizeof(dwNewLong));
```

Reply:
No, this is explcitly the type by intention.